### ISSUE TITLE
false positive no-member on connexion.exceptions.*

### ISSUE THREAD

**Issue by daniellehanks** [PyCQA/pylint#5061]
### Bug description

```python
Accessing a completely valid class of connexion.exceptions (https://github.com/zalando/connexion/blob/main/connexion/exceptions.py) causes no-member error. The error is odd though, referencing werkzeug.exceptions. While werkzeug is installed as a dependency of connexion, it is nowhere in the chain of inheritance for this class.

Assume the following content is saved to foo.py:

"""Demonstrate pylint bug."""
import connexion

err = connexion.exceptions.BadRequestProblem()
```
```


### Configuration

_No response_

### Command used

```shell
pylint foo.py
```

### Pylint output

```shell
************* Module test
foo.py:4:6: E1101: Module 'werkzeug.exceptions' has no 'BadRequestProblem' member (no-member)
```

### Expected behavior

Your code has been rated at 10.00/10

### Pylint version

```shell
pylint 2.11.1
astroid 2.8.0
Python 3.9.0 (default, Nov 13 2020, 04:16:05) 
[Clang 12.0.0 (clang-1200.0.32.2)]
```

### OS / Environment

macOS Big Sur 11.6, bash

### Additional dependencies

astroid==2.8.0
attrs==21.2.0
certifi==2021.5.30
charset-normalizer==2.0.6
click==7.1.2
clickclick==20.10.2
connexion==2.9.0
Flask==1.1.4
idna==3.2
inflection==0.5.1
isodate==0.6.0
isort==5.9.3
itsdangerous==1.1.0
Jinja2==2.11.3
jsonschema==3.2.0
lazy-object-proxy==1.6.0
MarkupSafe==2.0.1
mccabe==0.6.1
openapi-schema-validator==0.1.5
openapi-spec-validator==0.3.1
platformdirs==2.3.0
pylint==2.11.1
pyrsistent==0.18.0
PyYAML==5.4.1
requests==2.26.0
six==1.16.0
toml==0.10.2
typing-extensions==3.10.0.2
urllib3==1.26.6
Werkzeug==1.0.1
wrapt==1.12.1

**Comment by clavedeluna** [comment]
This is reproducible after running `pip install connexion`
